webapp/i18n/i18n.jsx

```diff
@@ -3,6 +3,8 @@
 
 const es = require('!!file?name=i18n/[name].[ext]!./es.json');
 const pt = require('!!file?name=i18n/[name].[ext]!./pt.json');
+const fr = require('!!file?name=i18n/[name].[ext]!./fr.json');
+
```

Comment:
I think ESLint is complaining here.

One other thing can you Squash your history based as docs the indicate: http://docs.mattermost.com/developer/contribution-guide.html?highlight=squash#preparing-a-pull-request